packages/client-core/src/admin/components/Project/UpdateDrawer.tsx

```diff
@@ -68,17 +67,17 @@ const UpdateDrawer = ({ open, builderTags, onClose }: Props) => {
   const processing = useHookstate(false)
   const helmSetting = useFind(helmSettingPath).data.at(0)
 
-  const adminProjectState = useHookstate(getMutableState(ProjectState))
-  const adminProjects = adminProjectState.projects
-  const engineCommit = adminProjectState.builderInfo.engineCommit
+  const adminProjects = useFind('project').data
+  const engineCommit = useGet('builder-info', undefined).data?.engineCommit
```

Comment:
Instead of passing params as undefined, cant we made this 2nd argument as optional?

Reply:
yes